handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -790,6 +796,7 @@ private void wrapNonAppData(ChannelHandlerContext ctx, boolean inUnwrap) throws
                 out.release();
             }
         }
+        return false;
```

Comment:
@Scottmitch like discussed before I think we should factor this out in a separate PR.